app/src/main/java/org/oppia/android/app/fragment/BUILD.bazel

```diff
@@ -91,3 +91,18 @@ kt_android_library(
         "//third_party:androidx_appcompat_appcompat",
     ],
 )
+
+kt_android_library(
+    name = "injectable_bottom_sheet_dialog_fragment",
+    srcs = [
+        "InjectableBottomSheetDialogFragment.kt",
+    ],
+    visibility = ["//app:app_visibility"],
+    deps = [
+        ":fragment_component_factory",
+        "//app/src/main/java/org/oppia/android/app/view:view_component",
+        "//app/src/main/java/org/oppia/android/app/view:view_component_builder_injector",
+        "//app/src/main/java/org/oppia/android/app/view:view_component_factory",
+        "//third_party:androidx_appcompat_appcompat",
```

Comment:
```suggestion
        "//third_party:androidx_appcompat_appcompat",
        "//third_party:com_google_android_material_material",
```

Reply:
Updated thanks

Reply:
@rt4914 can you please resolve this comment I am not able to resolve it